lib/rules/web/console/base/login.xyaml

```diff
@@ -11,21 +11,12 @@ login_sequence:
     ref:
     - click_pre_login_button
     - pre_login_buttons_gone
-    - login_page_loaded
```

Comment:
The rule login_page_loaded is used to make sure the required elements are loaded successfully. I am not sure what will be caused by simply removing it. @akostadinov cloud you give some advices?

Reply:
Exactly, that action should make sure that login prompt has appeared. In case it is not working well now, then better fix it than remove. Also IIRC sometimes we need first click a button to reach login prompt, this also checks for it.

If it is removed, it is possible that we open the page and try to set username/password before the text boxes appeared (or button to click appeared).